src/utils/systemJsRendering.ts

```diff
@@ -1,12 +1,16 @@
 import Variable from '../ast/variables/Variable';
 
 export function getSystemExportStatement(exportedVariables: Variable[]): string {
```

Comment:
I changed this to not write a trailing `;` so it could be used in a wider variety of places. Let me know if I should create something similar to this and leave this function alone.

Reply:
Seems sensible!